### ISSUE TITLE
Can I go back to Realm 4.4.0 from 5.3.6.  My code will not build on 5.3.6.

### ISSUE THREAD

**Issue by MikeGranberry** [realm/realm-cocoa#6736]
!!! MANDATORY TO FILL OUT !!!

## Version of Realm and Tooling

Realm framework version: 5.3.6

Realm Object Server version: ?```
ProductName:	Mac OS X
ProductVersion:	10.15.6
BuildVersion:	19G2021

/Applications/Xcode.app/Contents/Developer
Xcode 11.7
Build version 11E801a

/usr/local/bin/pod
1.9.3
(not in use here)

/bin/bash
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin19)

/usr/local/bin/carthage
0.33.0
(not in use here)

/usr/bin/git
git version 2.24.3 (Apple Git-128)
```

**Comment by MikeGranberry** [comment]
I was able to do the "pod update" after point my my podfile to the 4.4.0 version with the following line in the podfile:
pod 'RealmSwift', '4.4.0'
The load of 4.4.0 occurred as I requested.  But now I have the same Build Errors in my previously clean code that I had after the 5.3.6 update.  I am not sure how to correct the Build system to keep from getting duplicate files and unfound Realm files.
